webform_copy_value.module

```diff
@@ -137,7 +141,34 @@ function webform_copy_value_submit($form, &$form_state) {
       db_query("UPDATE webform_copy_value set `source` = '{$form_state['values']['copy_value_of']}' where id = {$mapId}");
     }
   } else if($sourceFieldId == 0) {
-    db_query("DELETE from webform_copy_value where id = {$mapId}");
+    //db_query("DELETE from webform_copy_value where id = {$mapId}");
```

Comment:
Fixed this. This actually needs to be uncommented but with additional condition.
Thanks :)